next.config.js

```diff
@@ -3,21 +3,23 @@
 const { PHASE_DEVELOPMENT_SERVER } = require("next/dist/shared/lib/constants")
 const i18nConfig = require("./i18n.config.json")
 
+const BUILD_LANGS = (process.env.BUILD_LANGS || "").replaceAll(/\s/g, "").split(",").filter(item => item.length > 1)
+const LIMIT_CPU = (process.env.LIMIT_CPU || "").toLowerCase() === "true"
```

Comment:
Hm, will look into that... but yeah, that's why I repeated it... was having a hard time importing

Reply:
@pettinarip Gave this a shot, but was unsuccessful. Tried switching to a `.mjs` file and followed the structured suggested in the docs, and importing these values from `constants`. Build fails immediately, complaining about the module not being available. Tried both `import` and `require syntax` with both `.js` and `.mjs`, and both `@` shortcuts and `../` relative paths... can't seem to get any to work to avoid repeating this logic. Let me know if you have any suggestions.

Reply:
@wackerow thanks for the investigation here. I'll play a bit with it today/tomorrow and get back to you.

Reply:
Don't see this as a blocker, especially with the number of paths being built on the rise with more content now imported. We could break this out as a separate task to consolidate as a DRY export

Reply:
Agreed.